### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by TimmEDK**
Some new versions and path

### CODE REVIEW

File: src/manjaro.ipxe [hunk 1]
```diff
@@ -25,8 +24,8 @@ goto mirror_cfg
 
 :mirror_cfg
 set flavor_root ${flavor}
-iseq ${flavor} architect && set ${manjaro_version} 17.1.7 ||
-set manjaro_initrd http://sourceforge.net/projects/manjarolinux/files/release/${manjaro_version}/${flavor_root}/manjaro-${flavor}-${manjaro_version}-stable-x86_64.iso/download
+iseq ${flavor} architect && set ${manjaro_version} 17.1.10 ||
```
Comment (antonym) [inline src/manjaro.ipxe:27]: Looks like Architect is 17.1.9 still.

File: src/manjaro.ipxe [hunk 2]
```diff
@@ -9,13 +9,12 @@ goto ${menu}
...
 menu ${os} Installers
 item --gap Official Releases
-item 17.1.8 ${space} ${os} 17.1.8
+item 17.1.10 ${space} ${os} 17.1.10
 choose manjaro_version || goto manjaro_exit
 
 :manjaro_flavor
 menu ${os} ${manjaro_version} Installers
 item --gap Flavors
-item architect ${manjaro_version} Architect
```
Comment (antonym) [inline src/manjaro.ipxe:18]: Any reason for dropping?

**Review by antonym — CHANGES_REQUESTED** [review]
Great updates, any reason for dropping Architect?

File: src/manjaro.ipxe (hunk 2 above)
Comment (TimmEDK) [reply src/manjaro.ipxe:18]: Only because there aren't no 17.1.10 for that official

Comment (antonym) [reply src/manjaro.ipxe:18]: Yeah, Architect isn't always updated every version, that's why the iseq for arcitect is in place, if you  can change it to:

```
iseq ${flavor} architect && set ${manjaro_version} 17.1.9 ||
```
And add back in Architect we can see about merging it.

Thanks!

Comment (TimmEDK) [reply src/manjaro.ipxe:18]: Sorceforge has switched to osdn, which doesn't fetch the ISO in the script - even directly

Comment (antonym) [reply src/manjaro.ipxe:18]: Yeah, I see the issue, we'll probably need to dig into that as it's currently using Sweden mirrors which aren't ideal.  It's broken currently anyways so we can move it in and we can open a bug for optimizing that mirror.

**Review by antonym — APPROVED** [review]

**Pull request merged**